docs/package.json

```diff
@@ -40,7 +40,7 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../.prettierignore",
 		"rebuild": "npm run clean && npm run build",
 		"serve": "docusaurus serve",
-		"serve-with-azure-emulation": "swa start",
+		"serve-with-azure-emulation": "swa start --config-name ff-doc-site",
```

Comment:
Unclear from the CLI docs what the policy is for a config file with only a single "configuration" entry, but this makes things more explicit.